### ISSUE TITLE
Unable to click "save" button on features

### ISSUE THREAD

**Issue by sschweigert** [mapbox/geojson.io#824]
In the attached image I am unable to click on the "save" button. I have confirmed this behavior on several different browsers and computers. This was working just yesterday, so I think a recent change may have broken this.

![image](https://user-images.githubusercontent.com/9129142/221719143-d895bf06-9b71-4723-85b6-68534ec8f4ec.png)

**Comment by sschweigert** [comment]
In firefox I get:
```
Uncaught TypeError: n is undefined
    M1 popup.js:51
    Ne d3.js:1120
    r helpers.js:90
[popup.js:51:6](https://geojson.io/src/lib/popup.js)
    M1 popup.js:51
    Ne d3.js:1120
    r helpers.js:90
```
Possibly related to https://github.com/mapbox/geojson.io/pull/819?

**Comment by marcbuettner** [comment]
same issue

**Comment by chriswhong** [comment]
Thanks for flagging this, working on a fix now.

**Comment by chriswhong** [comment]
Fixed in #825
